pkg/signal/signal.go

```diff
@@ -11,6 +11,11 @@ import (
 	"syscall"
 )
 
+const (
+	// DefaultStopTimeout is the timeout (in seconds) for the syscall signal used to stop a container.
+	DefaultStopTimeout = 10
```

Comment:
Thanks @cpuguy83. I tried to move the `DefaultStopTime` to container package. However, it caused a circular import issue in `runconfig/opts/parse.go` as `runconfig` is used by `container` package. Let me investigate further to see if there is a way to get around this issue.